pkgs/top-level/all-packages.nix

```diff
@@ -483,6 +483,12 @@ let
 
   aegisub = callPackage ../applications/video/aegisub {
     wxGTK = wxGTK30;
+    spellcheckSupport = config.aegisub.spellcheck or true;
+    automationSupport = config.aegisub.automation or true;
+    openalSupport     = config.aegisub.openal or false;
+    alsaSupport       = config.aegisub.alsa or true;
+    pulseaudioSupport = config.aegisub.pulseaudio or true;
+    portaudioSupport  = config.aegisub.portaudio or false;
```

Comment:
No special reason. I prefer without the "Support" suffix on them.

Also, about the failure, I don't know why. Here in my local machine it builds fine.